core/src/test/scala/org/broadinstitute/dsde/rawls/monitor/AvroUpsertMonitorSpec.scala

```diff
@@ -653,21 +646,20 @@ class AvroUpsertMonitorSpec(_system: ActorSystem)
           msg
             .attributes("errorMessage")
             .contains(
-              "Successfully updated 1000 entities; 1 updates failed. First 100 failures are: test-type this-entity-does-not-exist not found"
+              "Successfully updated 1000 entities; 1 updates failed."
```

Comment:
Quicksilver doesn't report exactly which reference isn't found, and I'm ok with that. It's not a huge risk but NOT echoing the user input is a good security stance to protect against XSS, etc.